db/reviews_data.csv

```diff
@@ -0,0 +1,41 @@
+rating,comment
+4,"Bestfriend time brightened my day. It was also promises to make shooting daytime and nighttime action simpler and professional-looking, providing the budding photographer with new and enticing reasons to hone their skills."
+4,"Ada best seller pack brightened my day. It was also promises to make shooting daytime and nighttime action simpler and professional-looking, providing the budding photographer with new and enticing reasons to hone their skills."
+1,Cup of rainbows design does not hold up on rugged use and did not reduce fatique. 
```

Comment:
It looks like some of these comments are surrounded by strings and some are not. Does this matter? Just wanna check.

Reply:
I think its fine with or without, saves good in db.